SConscript

```diff
@@ -464,7 +464,7 @@ def buildLibs(projectDir):
         '00std': [                                                              ],
         '01bsp': [                                                      'libbsp'],
         '02drv': [                                         'libdrivers','libbsp'],
-        '03oos': ['libopenstack','libopenapps','libkernel','libdrivers','libbsp'], # this order needed for mspgcc
```

Comment:
this is very strange, as the order is needed by mspgcc. What changing this order?